tests/test_ramp_fitting.py

```diff
@@ -1194,7 +1194,35 @@ def test_new_saturation():
                       [[0.03073696, 0.        , 0.]]])
     np.testing.assert_allclose(cerr, check, tol, tol)
 
-
+def test_contiguous_jumps():
```

Comment:
This test doesn't appear to test anything.  It simply opens a file, modifies it, then writes some arrays to another file.

Reply:
I added the test to do some debugging. I've removed it.